src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/PackageSourceTelemetry.cs

```diff
@@ -343,127 +363,82 @@ public Totals(int requests, long bytes, TimeSpan duration)
 
         internal class Data
         {
-            internal ResourceData Metadata { get; }
-            internal ResourceData Nupkg { get; }
+            internal object _lock;
+            internal Dictionary<string, (int count, TimeSpan duration)> Resources { get; }
+            internal HttpData Http { get; }
+            internal int NupkgCount { get; set; }
```

Comment:
Checking my understanding here... it looks like we used to track a Nupkg as a unit, but now we are merging the Count and Size of all Nupkgs into one metric. So we no longer can know the specific nupkg size that was being transferred, but only have the summed size?

Reply:
`PackageSourceTelemetry` always aggregated/accumulated numbers, so unless there's only a single nupkg, we won't know the size of each individually.